### PR TITLE
feat: (speaker-stats) add speaker stats feature to native

### PR DESCRIPTION
**Pull request by andioltean**
add speaker stats feature to mobile version of app
used new /native and /web folder structure for SpeakerStats* component
used new JitsiScreen modal

Screenshots
https://user-images.githubusercontent.com/5053543/137898438-14f07c4a-e709-4c0e-8b97-eea8ef9a6a4d.png
https://user-images.githubusercontent.com/5053543/137898446-1262285a-1cc6-40a1-95c5-04550688bf8a.png

### CODE REVIEW

File: react/features/speaker-stats/components/web/SpeakerStatsButton.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    _handleClick() {
+        const { dispatch, handleClick } = this.props;
+
+        if (handleClick) {
+            handleClick();
+
+            return;
+        }
+
+        sendAnalytics(createToolbarEvent('speaker.stats'));
```
Comment (quitrk) [inline react/features/speaker-stats/components/web/SpeakerStatsButton.js:31]: shouldn't we send analytics regardless of `handleClick` being truthy or not?

Comment (quitrk) [reply react/features/speaker-stats/components/web/SpeakerStatsButton.js:31]: i see you're sending it anyway in the `handleClick` so maybe remove it from there and just send it from here

**Review by quitrk — CHANGES_REQUESTED** [review]

File: react/features/speaker-stats/components/web/SpeakerStatsButton.js (hunk 1 above)
Comment (quitrk) [reply react/features/speaker-stats/components/web/SpeakerStatsButton.js:31]: nvm, i confused things here

**Review by quitrk — DISMISSED** [review]

**Comment by saghul** [comment]
Jenkins add to whitelist.

**Comment by saghul** [comment]
Jenkins please test this please.

File: react/features/speaker-stats/components/native/SpeakerStatsItem.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    isDominantSpeaker: boolean
+};
+
+const SpeakerStatsItem = (props: Props) => {
+    /**
+     * @inheritdoc
+     * @returns {ReactElement}
+     */
+    const dotColor = props.isDominantSpeaker
+        ? colors.success04 : colors.surface07;
```
Comment (Calinteodor) [inline react/features/speaker-stats/components/native/SpeakerStatsItem.js:40]: Please use BaseTheme here. More exactly BaseTheme.palette object.

**Review by Calinteodor — DISMISSED** [review]

**Review by Calinteodor — APPROVED** [review]

**Pull request merged**